GliaWidgets/SecureConversations/SecureConversations.UnreadMessageLoader.swift

```diff
@@ -0,0 +1,96 @@
+import Dispatch
+import GliaCoreDependency
+
+extension SecureConversations {
+    struct MessagesWithUnreadCount {
+        var messages: [ChatMessage]
+        var unreadCount: Int
+    }
+
+    /// Due to async nature of requests, we need to make sure
+    /// that `unread message count` and `transcript messages`
+    /// are delivered at once. This entity makes it possible.
```

Comment:
would we like to have it as doc or as comment?
```suggestion
    // Due to async nature of requests, we need to make sure
    // that `unread message count` and `transcript messages`
    // are delivered at once. This entity makes it possible.
```

Reply:
Doc for internal usage.